Gemfile

```diff
@@ -54,3 +54,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem "rubocop", "~> 1.12"
```

Comment:
I'd suggest moving this gem to the `development` group, since it's not required for any other env 😄

Reply:
Also, there is a complimentary gem called `rubocop-rails`, which has some extra checks/lints for rails projects 😄 I'd suggest to take a look at it

Reply:
Oh great , We will change it 👍